core/src/main/scala/com/nvidia/spark/rapids/tool/qualification/Qualification.scala

```diff
@@ -258,7 +258,7 @@ class Qualification(outputPath: String, numRows: Int, hadoopConf: Configuration,
       QualificationReportGenerator.generateDashBoard(outputDir, allAppsSum)
     }
     if (clusterReport) {
-      qWriter.writeClusterReport(allAppsSum)
```

Comment:
We will be generating cluster information in CSV format instead of JSON. However `writeClusterReport()` function will be still exists incase we need JSON format in future